cypress/e2e/3_Create_20_bounties.cy.ts

```diff
@@ -0,0 +1,271 @@
+describe('Alice tries to create 20 bounties', () => {
+  it('Create 20 bounties', () => {
+    let activeUser = 'alice';
+    cy.login(activeUser);
+    cy.wait(1000);
+
+    cy.create_bounty({
```

Comment:
instead of manually doing this 20 times can you do it in a for loop? I think doing this more programmatically will make the test easier to read